mizu/slots.py

```diff
@@ -69,12 +69,14 @@ def update_slot_status():
             return bad_params('No item with ID {} is present in the system'.format(item_id))
 
     if 'count' in body:
```

Comment:
```suggestion
    if 'count' in body and body['count'] is not None:
```

Reply:
we still want to update the slot's count if passed a `None` value, though.

Reply:
Ah yeah, didn't line up the `updates['count'] = count` line in my head, thought it was outside the if